Cannot install yaml-front-matter

```bash
$ npm install --save yaml-front-matter
npm ERR! Error: ENOENT, chmod '/Users/glavin/Documents/Project Dev/atom-beautify/node_modules/yaml-front-matter/node_modules/js-yaml/bin/js-yaml.js'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Darwin 14.0.0
npm ERR! command "/Users/glavin/.nvm/v0.10.32/bin/node" "/Users/glavin/.nvm/v0.10.32/bin/npm" "install" "--save" "yaml-front-matter"
npm ERR! cwd /Users/glavin/Documents/Project Dev/atom-beautify
npm ERR! node -v v0.10.32
npm ERR! npm -v 1.4.28
npm ERR! path /Users/glavin/Documents/Project Dev/atom-beautify/node_modules/yaml-front-matter/node_modules/js-yaml/bin/js-yaml.js
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! error rolling back Error: Refusing to delete: /Users/glavin/Documents/Project Dev/atom-beautify/node_modules/.bin/js-yaml not in /Users/glavin/Documents/Project Dev/atom-beautify/node_modules/yaml-front-matter
npm ERR! error rolling back     at clobberFail (/Users/glavin/.nvm/v0.10.32/lib/node_modules/npm/lib/utils/gently-rm.js:57:12)
npm ERR! error rolling back     at next (/Users/glavin/.nvm/v0.10.32/lib/node_modules/npm/lib/utils/gently-rm.js:43:14)
npm ERR! error rolling back     at /Users/glavin/.nvm/v0.10.32/lib/node_modules/npm/lib/utils/gently-rm.js:52:12
npm ERR! error rolling back     at Object.oncomplete (evalmachine.<anonymous>:107:15)
npm ERR! error rolling back  { [Error: Refusing to delete: /Users/glavin/Documents/Project Dev/atom-beautify/node_modules/.bin/js-yaml not in /Users/glavin/Documents/Project Dev/atom-beautify/node_modules/yaml-front-matter]
npm ERR! error rolling back   code: 'EEXIST',
npm ERR! error rolling back   path: '/Users/glavin/Documents/Project Dev/atom-beautify/node_modules/.bin/js-yaml' }
npm ERR! not ok code 0
```

I think https://github.com/dworthen/js-yaml-front-matter/blob/master/package.json#L25 is the culprit and should be removed. 

I already have `js-yaml` as a dependency, so this conflict may be causing this problem in my case.

--- a/package.json
+++ b/package.json
@@ -26,8 +26,7 @@
     "should": "0.6.3"
   },
   "bin": {
-    "yaml-front-matter": "bin/js-yaml-front.js",
-    "js-yaml": "node_modules/js-yaml/bin/js-yaml.js"
+    "yaml-front-matter": "bin/js-yaml-front.js"
   },
   "scripts": {
     "test": "mocha -u bdd --reporter spec",

